kolibri/core/assets/src/mixins/commonCoreStrings.js

```diff
@@ -1027,7 +1027,7 @@ export const coreStrings = createTranslator('CommonCoreStrings', {
     context: 'Section header for the list of resources that are related to the current resource',
   },
   doNotShowAgain: {
-    message: "Don't show this again",
+    message: "Don't show this message again",
```

Comment:
The message here now seems to mismatch with the context - the context is about resources, not messages.